.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+
+jobs:
+  fedora:
+    name: Linux Fedora (Release, GCC, cxx11)
+    runs-on: ubuntu-latest
+    container: dynawo/dynawo-ci:latest
+    strategy:
+      fail-fast: false
+    env:
+      DYNAFLOW_LAUNCHER_BUILD_TYPE: Release
```

Comment:
Should be Debug I think if we build tests

Reply:
Changed build type to Debug